src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -399,7 +399,13 @@ public void close(Collection<TopicPartition> partitions) {
   }
 
   @Override
-  public void setIsStoppedToTrue() {}
+  public void stop() {
+    try {
+      streamingIngestClient.close();
```

Comment:
QQ - Why not call `closeAll()` here? It closes/clears couple of more resources ...

Reply:
Thats a good point, closeAll will also close Channels and clear the cache. 
close client is already closing all channels so I dont see any issue with using closeAll here. 
@sfc-gh-tzhang do you see any issues with closing channels first vs closing client directly?

Reply:
One extra thing if you call close channel first is that we will check and throw exception if not everything is committed; calling close on client directly will just mark all channels as closed. For KC I think it's fine to just close the client.